### DISCUSSION TITLE
allow user to resize columns in the Query Tab

### DISCUSSION THREAD

**Discussion by capslock66** [Kong/insomnia#4962] [Ideas]
Query tab contains 2 columns : names and values.
Names is often more shorter than values.
Odata sample : $filter = groupby((StatusKind),aggregate(StatusKind with countdistinct as total))
So It's better if we can resize the 2 columns

**Comment by capslock66** [comment]
Any news ?

**Comment by gschier** [comment]
I think this would be a great feature to have. It's just fairly low priority compared to everything else. If you, or anyone else, would like to contribute a PR for it, I'd be happy to provide guidance.

**Comment by dimitropoulos** [comment]
This is a really great ticket and I want this one too!  We're moving all feature requests to GitHub Discussions
